docs/src/library/qpu.md

```diff
@@ -18,4 +18,6 @@ get_excluded_connections
 path_search
 get_adjacency_list
 get_qubits_distance
+is_native_instruction
```

Comment:
I'm not sure if these should be part of the public interface. We are only implementing them for some of the connectivities

Reply:
Are you only referring to `is_native_instruction` and `is_native_circuit` or are you also referring to `path_search`, `get_adjacency_list`, and `get_qubits_distance`?

Reply:
I would at least export `is_native_circuit` since it is useful if you're trying to make sure that the gates in your circuit won't be replaced by a larger number of gates during transpilation.

Reply:
yes I was referring to `is_native_instruction` and `is_native_circuit`. Ok if we add the NotImplementedErrors for AbstractConnectivity, it will be a catch-all for undefined methods